speechbrain/nnet/quantisers.py

```diff
@@ -0,0 +1,107 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class GumbelVectorQuantizer(nn.Module):
```

Comment:
Where did you get your inspiration from ? If the code is taken from Fairseq or HF, we must mention it.

Reply:
will do

Reply:
done